lms/djangoapps/certificates/api.py

```diff
@@ -452,3 +452,41 @@ def emit_certificate_event(event_name, user, course_id, course=None, event_data=
 
     with tracker.get_tracker().context(event_name, context):
         tracker.emit(event_name, event_data)
+
+
+def get_certificate_header():
+    """
+    Return data to be used in Certificate Header,
+    data returned should be customized according to the microsite settings
+    """
+    data = dict(
+        logo_src=get_logo_url(),
+        logo_url=branding_api.get_base_url(),
+    )
+
+    return data
+
+
+def get_certificate_footer():
```

Comment:
As with above, let's rename this to something like 'get_certificate_footer_context' or 'get_certificate_footer_data'